### DISCUSSION TITLE
doris什么时候实现binlog功能？

### DISCUSSION THREAD

**Discussion by shiliming** [apache/doris#45420] [Ideas]
在2024年的roadmap上有 Support data binlog，但迟迟没有实现。
现在还有计划吗？等这个功能好久了。
如果不支持doris自身的cdc，那实时数仓从何谈起？
只是实时从业务数据库获取数据到ods层而已，后续的数据加工都做不到实时数据。

**Comment by ixzc** [comment]
you can submit your requirements here
https://ask.selectdb.com
